app/main.py

```diff
@@ -1,6 +1,38 @@
+import os
+
+import requests
+
+API_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    """
+    Fetches current weather data for Paris city and displays it in the console.
+    """
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        raise ValueError("API_KEY environment variable is not set")
+
+    params = {"key": api_key, "q": CITY}
+
+    response = requests.get(API_URL, params=params)
+    response.raise_for_status()
+
+    data = response.json()
+
+    location = data["location"]
+    country = location["country"]
+    current = data["current"]
+    local_time = location["localtime"]
+    current_temp = current["temp_c"]
+    current_conditions = current["condition"]["text"]
+
+    print(f"Performing request to Weather API for city {CITY}...")
+    print(
+        f"{CITY}/{country} {local_time} Weather: {current_temp} Celsius, "
+        f"{current_conditions}"
+    )
```

Comment:
This doesn't follow the Single Responsibility Principle. You could split it into two functions and use a try-except block for each. Also, consider changing the naming of the response and other variables. Thanks!